src/config/layerx.json

```diff
@@ -65,7 +70,7 @@
     "available": false
   },
   "hydra": {
-    "available": false
+    "available": true
```

Comment:
@brianshattuck why this is changed from false to true?

Reply:
my branch was behind dev2 branch for a while, and that's why